ts/client/src/client.ts

```diff
@@ -517,6 +517,7 @@ export class OpenBookV2Client {
       accountIndex,
       name,
       owner.publicKey,
+      null,
```

Comment:
Rather than null, we could add an arg to the function which is the delegateAccount

Reply:
yes will do